src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -1,57 +1,85 @@
 import React from 'react';
 import { withStyles } from '@material-ui/core/styles';
 
+const topAligned = {
+  '--trans-y': 'var(--tip-size)',
+  '&::before': { bottom: '100%', borderBottomColor: 'var(--bg-color)' },
+  '&.xright': {
+    '--trans-x': 'calc(-100% + 2 * var(--tip-size))',
+    '&::before': { right: 'var(--tip-size)' },
+  },
+  '&.xcenter': {
+    '--trans-x': '-50%',
+    '&::before': { left: 'calc(50% - var(--tip-size))' },
+  },
+  '&.xleft': {
+    '--trans-x': 'calc(-2 * var(--tip-size))',
+    '&::before': { left: 'var(--tip-size)' },
+  },
+};
 const styles = {
   tooltip: {
+    '--bg-color': 'rgba(0, 0, 0, .8)',
+    '--tip-size': '6px',
     position: 'absolute',
     padding: '6px',
-    background: 'rgba(0, 0, 0, .8)',
+    background: 'var(--bg-color)',
     color: 'white',
     borderRadius: '4px',
     pointerEvents: 'none',
+    transform: 'translate(var(--trans-x), var(--trans-y))',
+    '& a': {
+      pointerEvents: 'auto',
```

Comment:
We need to exclude anchors from our previous `pointerEvents: 'none'`, otherwise they wouldn't be clickable. And we need the last one in order to hover datapoints under the tooltip.